website/versioned_docs/version-0.62/releases.md

```diff
@@ -0,0 +1,25 @@
+---
+id: version-0.62-releases
+title: Release Strategy (vnext)
+original_id: releases
+---
+
+> This page is hanging, see [Issue #59](https://github.com/microsoft/react-native-windows-samples/issues/59).
```

Comment:
@kikisaints  - can you add a Releases sidebar item so this page is not hanging?

Reply:
added to the sidebar! removed the link to the issue on that page too.